stratum/hal/lib/tdi/dpdk/dpdk_switch.h

```diff
@@ -104,9 +107,9 @@ class DpdkSwitch : virtual public SwitchInterface,
 
   // Factory function for creating the instance of the class.
   static std::unique_ptr<DpdkSwitch> CreateInstance(
-      DpdkChassisManager* chassis_manager,
-      TdiSdeInterface* sde_interface,
-      const std::map<int, TdiNode*>& device_id_to_tdi_node);
+      DpdkChassisManager* dpdk_chassis_manager,
```

Comment:
Same comment as before. You're undoing work I did for a specific reason, and you're distracting the reader from the changes that matter.